corehq/apps/sms/management/commands/migrate_sms_to_sql.py

```diff
@@ -0,0 +1,50 @@
+from corehq.apps.sms.models import SMSLog, SMS
+from custom.fri.models import FRISMSLog
+from dimagi.utils.couch.database import iter_docs
+from django.core.management.base import BaseCommand
+from optparse import make_option
+
+
+class Command(BaseCommand):
+    args = ""
+    help = ("Migrates SMSLog to SMS")
+    option_list = BaseCommand.option_list + (
+        make_option("--balance_only",
+                    action="store_true",
+                    dest="balance_only",
+                    default=False,
+                    help="Include this option to only run the balancing step."),
+    )
+
+    def get_sms_couch_ids(self):
+        result = SMSLog.view(
+            'sms/by_domain',
+            include_docs=False,
+            reduce=False,
+        ).all()
+        return [row['id'] for row in result if row['key'][1] == 'SMSLog']
+
+    def run_migration(self):
+        count = 0
+        ids = self.get_sms_couch_ids()
+        total_count = len(ids)
+        for doc in iter_docs(FRISMSLog.get_db(), ids):
+            couch_sms = FRISMSLog.wrap(doc)
```

Comment:
It was the easiest way to do it.  SMSLog can't sync the FRI fields because the attributes aren't defined on SMSLog, and I didn't want to pull from the json because then I'd have to apply the defaults manually.  So since FRISMSLog is the superset of all fields anyway, this takes care of FRISMSLog and SMSLog entries.